db/seeds.rb

```diff
@@ -1,30 +1,34 @@
 # frozen_string_literal: true
 
-# This file should contain all the record creation needed to seed the database
-# with its default values.
-# The data can then be loaded with the rails db:seed command (or created
-# alongside the database with db:setup).
-#
-# Examples:
-#
-#  movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#  Character.create(name: 'Luke', movie: movies.first)
-
-url_path_method = lambda do |user|
-  V2::UsersController.resource_url_path(user)
+# Create users.
+url_path_method = lambda do |resource|
+  V2::UsersController.resource_url_path(resource)
 end
 %w(ada bob).each do |name|
-  User.new(name: name, url_path_method: url_path_method).save
+  User.new(name: name,
+           email: "#{name}@example.com",
+           url_path_method: url_path_method).save
+end
+
+# Create organizations.
+url_path_method = lambda do |resource|
+  V2::OrganizationsController.resource_url_path(resource)
+end
+Organization.new(name: 'All Users', url_path_method: url_path_method).save
```

Comment:
Is this just an example or do we want to have an organization that all users are part of?